swan-lake/by-example/ftp-listener-format-callbacks/content.jsx

```
@@ -0,0 +1,336 @@
+                width="20"
+                height="20"
+                fill="#3ad1ca"
+                className={`${
+                  btnHover[0] ? "btnArrowHover" : "btnArrow"
+                } bi bi-arrow-right`}
+                viewBox="0 0 16 16"
+                onMouseEnter={() => updateBtnHover([true, false])}
+                onMouseOut={() => updateBtnHover([false, false])}
+              >
+                <path
+                  fill-rule="evenodd"
```

ESLint

[error] Unknown property 'fill-rule' found, use 'fillRule' instead

(react/no-unknown-property)